src/frontends/paddle/docs/operation_mapping_flow.md

```diff
@@ -0,0 +1,13 @@
+# OpenVINO Paddle Frontend operator enabling Flow
+
+1. Declare `CreatorFunction` for the Paddle operator and register it to the map in `src/op_table.cpp`.
+   * The map is retrived from:
+
+     https://github.com/openvinotoolkit/openvino/blob/7d5e0abcaa03703de9918ece2115e6ea652c39e0/src/frontends/paddle/src/op_table.cpp#L106
+
+2. Implement the operator mapper `CreatorFunction` in path `src/op/`.
```

Comment:
I am not sure that I am as an external user will be able to implement the operator without the example.

Reply:
updated